### PR TITLE
Fixed a syntax issue

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.7+ |
| Fixed tickets | - |

Hopefully this will solve the following issue:

syntax_issue

### CODE REVIEW

File: components/http_foundation/introduction.rst [hunk 1]
```diff
@@ -255,8 +255,8 @@ by using the following methods:
 :method:`Symfony\\Component\\HttpFoundation\\Request::getEncodings`
     Returns the list of accepted encodings ordered by descending quality.
 
-  .. versionadded:: 2.4
-      The ``getEncodings()`` method was introduced in Symfony 2.4.
+.. versionadded:: 2.4
+    The ``getEncodings()`` method was introduced in Symfony 2.4.
```
Comment (xabbuh) [inline components/http_foundation/introduction.rst]: I think you should simply indent this with four spaces (like the sentence before).

Comment (javiereguiluz) [inline components/http_foundation/introduction.rst]: According to platform.sh, this change fixes the issue: http://pr-5822-6qmocelev2lwe.eu.platform.sh/components/http_foundation/introduction.html#accessing-accept-headers-data

Comment (xabbuh) [inline components/http_foundation/introduction.rst]: Not really, the `versionadded` directive is now rendered after the definition list.

Comment (javiereguiluz) [inline components/http_foundation/introduction.rst]: Fixed. Thanks.

**Comment by xabbuh** [comment]
Thanks Javier.

**Pull request closed without merging**